### PR TITLE
Howto guide

### PR DESCRIPTION
**Pull request by torsava**
Hi!
I think I have fairly complete howto guide that can be put on the PortingDB. If you can spare some time, I'd love some criticism, @encukou, @hroncok.

Later I will also refresh the text on the top of the index/hello page in light of the howto-guide, so-far I've just added a monkey-patched a link to the guide there so we can first sort out the contents thereof.

### CODE REVIEW

**Comment by torsava** [comment]
Here is the [howto page already rendered](http://htmlpreview.github.io/?https://github.com/torsava/copy/blob/master/So%20you%20want%20to%20contribute_%20%E2%80%93%20Python%203%20Porting%20Database.html), so you can take a look at the rusult easier.

**Comment by torsava** [comment]
I was also wondering whether I should somehow mention a link to `Join Fedora packagers` wiki page and how to create an account on RHBZ, but ultimately decided agaist it. If you think it should be there, I'll try to work it in.

File: portingdb/templates/howto.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        </p><p>
+            To let us know, edit the <i>Blocks</i> field of the Bugzilla report and add the string <b>PY3PATCH-AVAILABLE</b> (you can use space as a separator). That way it will appear in our <a href="https://bugzilla.redhat.com/show_bug.cgi?id=1312032">tracking bug</a> for packages with an available patch. If need be, we'll review your patch and push it for the maintainer.
+        </p>
+        <h3>Fame and Glory</h3>
+        <p>
+            After you've done the deed, you deserve your reward! Sadly, the hand of the King's daughter and half of all the kingdom has recently become unavailable (may vary upon location, ask your nearest royal family), but fret not, you can get a <i>badge</i> of honour!
+        </p><p>
+            Visit the <a href="#TODO">wiki page for badges</a>, add a row with your <i>FAS username</i> and post a link to the Bugzilla report you helped resolve. You can get 3 badges: for your 1st, 5th and also 10th resolved package!
```
Comment (encukou) [inline portingdb/templates/howto.html]: What's the status here?

File: portingdb/templates/howto.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        </p>
+        <h3>Make it Happen</h3>
+        <p>
+            When you submit your patch to the associated Bugzilla report (use the <i>"Add an attachment"</i> button), be sure to ask the assgned packager to review and push it. However, as maintainers can be unresponsive for long periods of time, let us know about your patch as well so we can push it if the maintainer is unable.
+        </p><p>
+            To let us know, edit the <i>Blocks</i> field of the Bugzilla report and add the string <b>PY3PATCH-AVAILABLE</b> (you can use space as a separator). That way it will appear in our <a href="https://bugzilla.redhat.com/show_bug.cgi?id=1312032">tracking bug</a> for packages with an available patch. If need be, we'll review your patch and push it for the maintainer.
+        </p>
+        <h3>Fame and Glory</h3>
+        <p>
+            After you've done the deed, you deserve your reward! Sadly, the hand of the King's daughter and half of all the kingdom has recently become unavailable (may vary upon location, ask your nearest royal family), but fret not, you can get a <i>badge</i> of honour!
```
Comment (encukou) [inline portingdb/templates/howto.html]: Unfortunately, not everyone will understand this cultural reference :(

Comment (torsava) [inline portingdb/templates/howto.html]: It's not only Czech, but also anglophone (I checked). And if someone misses it, I think they can read on.

File: portingdb/templates/howto.html (hunk 1 above)
Comment (torsava) [inline portingdb/templates/howto.html]: Ticket is idle for 3 weeks, I'm not sure how to speed it up, but Miro should be back at work tomorrow or after that.

https://fedorahosted.org/fedora-badges/ticket/457

**Comment by torsava** [comment]
As advertised previously, here are a few some modifications of the Hello page to tie in with the Howto page.

File: data/config.yaml [hunk 3]
```diff
@@ -21,20 +21,21 @@ main-instructions: |
...
+    <div style="padding-top: 7pt;">
         Read me: <a href="$HOWTO_LINK">So you want to contribute?</a><br/>
+    </div>
+    <div style="padding-top: 7pt;">
         If you would like to jump in and help port, do this:
         <ul>
             <li>
-                If you have a specific project in mind that you would like to port,
-                click on it and follow instructions there.
-                (You can use your browser – Ctrl+F – to find your package on this page.)
```
Comment (encukou) [inline data/config.yaml]: Did you consider the [`<kbd>`](https://developer.mozilla.org/en/docs/Web/HTML/Element/kbd) tag?

File: portingdb/templates/howto.html (hunk 1 above)
Comment (encukou) [inline portingdb/templates/howto.html]: This is blocking the PR from being merged.

File: data/config.yaml (hunk 3 above)
Comment (torsava) [inline data/config.yaml]: Well, I think people capable of porting know about Ctrl+F so I thought shorter would be better.

File: portingdb/templates/howto.html (hunk 1 above)
Comment (torsava) [inline portingdb/templates/howto.html]: Absolutely, sorry, I could have mention it in the comment.

**Comment by torsava** [comment]
Removed section _Glory and Fame_ that was blocking the merge due to the badges not being ready yet.

**Pull request merged**